_data/tags_doc.yml

```diff
@@ -1,9 +0,0 @@
-allowed-tags:
```

Comment:
This is for defining tags (i.e., labels) that can be added to a page's front matter and then show up at the bottom of the page. The idea is to let the user click through to a list of all pages related to a tag.  

I'm ok with removing this for now, though I'll likely want to get this functionality back at some point. For my future reference: http://idratherbewriting.com/documentation-theme-jekyll/mydoc/mydoc_tags.html

Can you also remove the `{% assign projectTags = site.data.tags_doc.allowed-tags %}` line in `/_includes/custom/conditions.html`?